source/slang/hlsl.meta.slang

```diff
@@ -5062,6 +5097,7 @@ matrix<T, M, N> transpose(matrix<T, N, M> x)
 __generic<T : __BuiltinIntegerType, let N : int, let M : int>
 __target_intrinsic(hlsl)
 __target_intrinsic(glsl)
+__target_intrinsic(spirv, "OpTranspose resultType resultId _0")
```

Comment:
It seems OpTranspose is not defined integer matrices in spirv, same for other matrix op codes.